administrator/components/com_patchtester/language/en-GB/en-GB.com_patchtester.sys.ini

```diff
@@ -7,3 +7,17 @@ COM_PATCHTESTER="Joomla! Patch Tester"
 COM_PATCHTESTER_COULD_NOT_INSTALL_OVERRIDES="Could not install the template overrides for the following templates: %s"
 COM_PATCHTESTER_COULD_NOT_REMOVE_OVERRIDES="Could not remove the template overrides for the following templates: %s"
 COM_PATCHTESTER_XML_DESCRIPTION="Component for pull request management testing"
+COM_PATCHTESTER_GOTO="Go to"
+COM_PATCHTESTER_LINK="Component"
+COM_PATCHTESTER_OPTIONS="Options"
+COM_PATCHTESTER_INSTALL_TEXT="Patch Tester Install script"
+COM_PATCHTESTER_POSTFLIGHT_DISCOVER_INSTALL_TEXT="Patch Tester postflight discover install script"
+COM_PATCHTESTER_POSTFLIGHT_INSTALL_TEXT="Patch Tester postflight install script"
+COM_PATCHTESTER_POSTFLIGHT_UNINSTALL_TEXT="Patch Tester postflight uninstall script"
+COM_PATCHTESTER_POSTFLIGHT_UPDATE_TEXT="Patch Tester postflight update script"
+COM_PATCHTESTER_PREFLIGHT_DISCOVER_INSTALL_TEXT="Patch Tester preflight discover install script"
+COM_PATCHTESTER_PREFLIGHT_INSTALL_TEXT="Patch Tester preflight install script"
+COM_PATCHTESTER_PREFLIGHT_UNINSTALL_TEXT="Patch Tester preflight uninstall script"
+COM_PATCHTESTER_PREFLIGHT_UPDATE_TEXT="Patch Tester preflight update script"
+COM_PATCHTESTER_UNINSTALL_TEXT="Patch Tester Uninstall script"
```

Comment:
Is this debugging code?

Reply:
I don't believe I added them, they may have been legacy from my initial install of the Patch Tracker when I set about trying to fix the original issue

Reply:
In that case you can remove the lines from your file and commit and push them. They will be gone here as well.